File: Data Structures/Trees/Inorder Traversal.java
/* Contributer : github.com/Shaurya9619
	   Email : shaurya96gairola@gmail.com
	*/
static void inOrder(Node root) {
        if(root!=null){
           
        inOrder(root.left);
        System.out.print(root.data+"\t");
        inOrder(root.right);
    }
    
}

File: Data Structures/Trees/Inorder Traversal.cpp
/* Contributer : github.com/Shaurya9619
	   Email : shaurya96gairola@gmail.com
	*/
void inOrder(node* root) {
    
    if(root != NULL)
    {
        inOrder(root->left);
        cout << root->data <<" ";
        inOrder(root->right);

    }

}
